index.html

```diff
@@ -490,6 +490,12 @@ <h3>Proofs</h3>
 [[XMLSCHEMA11-2]] combined date and time string.
           </dd>
 
+          <dt>expires</dt>
+          <dd>
+The date and time the proof expires SHOULD be included and, if included, MUST be
+specified as an [[XMLSCHEMA11-2]] combined date and time string.
```

Comment:
```suggestion
An OPTIONAL property that conveys the date and time that a proof expires that MUST be
specified as an [[XMLSCHEMA11-2]] combined date and time string.
```

Reply:
This PR does not address the issue of the difference (or not) between expires and validFrom. This needs to be clarified